be/src/agent/heartbeat_server.cpp

```diff
@@ -131,6 +131,12 @@ void HeartbeatServer::heartbeat(THeartbeatResult& heartbeat_result, const TMaste
         }
         heartbeat_result.backend_info.__set_reboot_time(reboot_time);
     }
+
+    // Temporary solution for modifying the default value of disable_column_pool in lower versions
+    // to avoid behavior changes.
+    if (!config::disable_column_pool && master_info.run_mode == TRunMode::SHARED_DATA) {
+        config::disable_column_pool = true;
```

Comment:
at least add one line log, indicate that the configuration is turned on explicitly in shared data mode.

Reply:
ok